### DISCUSSION TITLE
Cannot find calendar.txt

### DISCUSSION THREAD

**Discussion by bukshee** [helix-editor/helix#7522] [Troubleshooting] [answered]
### Summary

I tried to open the file ~/Documents/notes/calendar.txt

But autocomplete does not find it:
:o ~/Documents/notes/cal&lt;TAB&gt;

All other files are visible in this folder, even other txt files, just this one is missing. If I manually enter the whole name and hit Enter, the file opens fine. Just not visible for autocomplete.

If I rename the file to calendar.txt.txt, autocomplete finds it.

It is a regular file:
notes » ls -l calendar.txt
-rw-r--r-- 1 adi adi 649 Jul  2 19:14 calendar.txt

### Reproduction Steps

 

I tried this:

1. `hx`

I expected this to happen:

Instead, this happened:

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

Manjaro Linux amd64

### Terminal Emulator

alacritty 0.12.0 (5a728195)

### Helix Version

helix 23.05

**Comment by bukshee** [comment]
The fuzzy-finder (space + f) won't find it either. However if I copy the file to other directories it finds it.

**Comment by the-mikedavis** [comment, marked as answer]
Does the `~/Documents/notes` directory have a `.gitignore` that ignores `calendar.txt`? Ignored files are excluded from the file picker (`<space>f`) and from `:open` completion by default.

**Reply by bukshee** [reply]
Yes. You are right. The file is there. Sorry my bad!
